torch/nn/quantized/modules/channelshuffle.py

```diff
@@ -0,0 +1,22 @@
+import torch
+
+class ChannelShuffle(torch.nn.ChannelShuffle):
+    def __init__(self, groups: int):
```

Comment:
is this constructor needed? since it just passes through the arguments

the base-derived constructor would be sufficient?

Reply:
Thanks. They are removed.